src/main/java/com/microsoft/cse/helium/app/Constants.java

```diff
@@ -16,6 +16,10 @@
   public static final String WEB_INSTANCE_ROLE = "WEBSITE_ROLE_INSTANCE_ID";
   public static final Integer MAX_DEGREE_PARALLELISM = 2;
 
+  public static final String AUTH_TYPE = "AUTH_TYPE";
+  public static final String KEY_VAULT_NAME = "KeyVaultName";
```

Comment:
env var is KEYVAULT_NAME

Reply:
Interesting, the Java Code had that from the very get go. Good catch. Is this documented some place? I will change it anyhow.

Reply:
I put that in initially. It can be changed to use code instead if that is the preference. I'm not sure c# was checking that at the time. I think it was relying on the API to enforce.

Reply:
@bartr 
actually, csharp also uses the constant KeyVaultName
        public const string KeyVaultName = "KeyVaultName";
        string kvName = Environment.GetEnvironmentVariable(Constants.KeyVaultName);